### ISSUE TITLE
md5 was modified

### ISSUE THREAD

**Issue by yimchengjie** [Eugeny/tabby#9332]
Version: 1.0.197
Platform: Windows x64 10.0.22631
Plugins: none
Frontend: xterm-webgl

use 'sz' command to download file, the file md5 was modified!

**Comment by shengminjie** [comment]
Yes, I have downloaded the latest version but it still hasn't been resolved. It has been a long time, such as sz, a 12MB compressed file that cannot be decompressed and is damaged.

**Comment by lidonghai11** [comment]
I have the same question. When will it be ready?

**Comment by Make-1** [comment]
I have the same question. When will it be ready?

**Comment by richelieu042** [comment]
I have the same question. When will it be ready?
But there is no problem when downloading using SFTP.
